src/library/scala/collection/Seq.scala

```diff
@@ -284,7 +284,7 @@ trait SeqOps[+A, +CC[_], +C] extends Any
     * @param    idx     the index to test
     * @return   `true` if this $coll contains an element at position `idx`, `false` otherwise.
     */
-  def isDefinedAt(idx: Int): Boolean = (idx >= 0) && (idx < length)
+  def isDefinedAt(idx: Int): Boolean = idx >= 0 && lengthIs > idx
```

Comment:
It would've been nice to have the alligators facing the same direction, but now you would need `lengthIs > idx && idx >= 0`.